modules/op-release-notes-1-9.adoc

```diff
@@ -433,4 +433,31 @@ With this update, {pipelines-title} General Availability (GA) 1.9.2 is available
 [id="fixed-issues-1-9-2_{context}"]
 == Fixed issues
 
-* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
\ No newline at end of file
+* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
+
+[id="release-notes-1-9-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.3
+
+With this update, {pipelines-title} General Availability (GA) 1.9.3 is available on {product-title} 4.10 in addition to 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-9-3_{context}"]
+== Fixed issues
+
+* This update fixes the performance issues for huge pipelines. Now, the CPU usage is reduced by 61% and the memory usage is reduced by 44%.
+
+* Before this update, a pipeline run would fail if a task did not run because of its `when` expressions. This update fixes the issue by preventing the validation of a skipped task result in pipeline results. Now, the pipeline result is not emitted and the pipeline run does not fail because of a missing result.
+
+* This update fixes the `pipelineref.bundle` conversion to the bundle resolver for the `v1beta1` API. Now, the conversion feature sets the value of the `kind` field to `Pipeline`.
+
+* Before this update, an issue in the {pipelines-shortname} Operator prevented the user from setting the value of the `spec.pipeline.enable-api-fields` field to `beta`. This update fixes the issue. Now, you can set the value to `beta` along with `alpha` and `stable` in the `TektonConfig` custom resource.
+
+* Before this update, when {pac} could not create a secret due to a cluster error, it would leak the temporary token on the GitHub check run, which is public. This update fixes the issue. Now, the secret token is no longer displayed on the GitHub checks when the creation of the secret fails.
+
+[id="known-issues-1-9-3_{context}"]
+== Known issues
+
+* There is currently a known issue with the *stop* option for pipeline runs in the {product-title} web console. The *stop* option from the *Actions* drop-down menu is not working as expected and does not cancell the pipeline run.
+
+* There is currently a known issue with upgrading to {pipelines-shortname} version 1.9.x due to a failing custom resource definition conversion.
```

Comment:
Do we have breaking change section as well :thinking: 
If yes it would be great if its added over there